ptx/sec_deriv_intro.ptx

```diff
@@ -1230,7 +1317,14 @@
           </caption>
           <!-- START figures/fig_piecewisecosx1.tex -->
           <image xml:id="img_piecewisecosx1" width="47%">
-            <description></description>
+            <shortdescription>
+              Graph of derivative of function that is <m>\cos(x)</m> for values of <m>x \leq \pi/2</m> and <m>x \gt \pi/2</m>.
```

Comment:
When you remove math tags, be careful not to use an unescaped less than or greater than. Also, be careful about ambiguity in language here: this could be read as saying that that this is the graph of the derivative of a function, where the function (not the derivative) is cos(x). (Also you are missing that it is zero after pi/2.)

You could try:
"The derivative is cos(x) until x=pi/2, and 0 after that. There is a sharp bend at pi/2."